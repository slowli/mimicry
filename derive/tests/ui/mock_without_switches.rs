use mimicry_derive::CallReal;

#[derive(CallReal)]
struct MyMock {
    not_a_switch: String,
    not_a_switch_either: u32,
}

fn main() {}
