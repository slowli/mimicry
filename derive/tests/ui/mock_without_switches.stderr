error: No fields of `RealCallSwitch` type. Please add such a field, or, if it's present, mark it with `#[mock(switch)]` attr
 --> tests/ui/mock_without_switches.rs:4:15
  |
4 |   struct MyMock {
  |  _______________^
5 | |     not_a_switch: String,
6 | |     not_a_switch_either: u32,
7 | | }
  | |_^
